include/mbedtls/config.h

```diff
@@ -769,6 +769,8 @@
 #define MBEDTLS_ECP_DP_BP512R1_ENABLED
 #define MBEDTLS_ECP_DP_CURVE25519_ENABLED
 #define MBEDTLS_ECP_DP_CURVE448_ENABLED
+#define MBEDTLS_ECP_DP_ED25519_ENABLED
+#define MBEDTLS_ECP_DP_ED448_ENABLED
```

Comment:
As long as only arithmetic is implemented but no cryptographic algorithm, should the Edwards curves be enabled by default?

* :heavy_plus_sign: If someone wants to use them now, it's nicer to have them enabled by default.
* :heavy_minus_sign: This increases the code size in the default build for something that most users can't use.
* :heavy_plus_sign: If the curves are not enabled by default, `tests/scripts/curves.pl` needs to be modified to take this into account.
* :heavy_minus_sign: This locks the API extensions — not a big deal since the API change (new enum entries, one new function) is very small and not controversial.

Reply:
I'm inclined to have them enabled by default.

We don't really care about code size in the default build, which is currently defined as "everything unless there's a good reason not to enable it" - I think it makes more sense to have Edwards curves even with partial support than NIST P-192 (which nobody uses and provides less than 128-bit of security), and we do have P-192 in the default build.

Regarding API extensions, as you say they're fairly small and straightforward.